en/docs/install-and-setup/setup/mi-setup/deployment/mi-cicd-overview.md

```diff
@@ -3,11 +3,13 @@
 Continuous integration(CI) and continuous deployment(CD) for Integration is a must for delivering changes more frequently and reliably. 
```

Comment:
Is this the correct place for the content ? this says mi-cicd right ?

Reply:
This is actually an Overview page for Integration CI/CD. But the page is named mi-cicd-overview. Do we need to change this? If we change this now, it will change the URL.

Reply:
We can change the URL but we'll have to do some redirects. No issues with the content and page title in my view.

Reply:
If the display name is just overview , I'm fine with it